app/test/frontend/handlers/_utils.dart

```diff
@@ -42,11 +43,13 @@ Future<shelf.Response> issueGet(
 Future<shelf.Response> issueHttp(
```

Comment:
What does this method do?

Reply:
Bridges to the shelf handlers without the overhead of the real HTTP connection and request parsing.
Now that we need the HTTP server anyway, I think we should migrate the code to call it directly.